src/Notification/helpers.js

```diff
@@ -0,0 +1,137 @@
+import { css } from 'styled-components';
+
+/**
+ * Get Alert Position
+ *
+ * @param {string} placement - Placement of the notification on the screen.
+ *
+ */
+
+export function getNotificationPosition(placement) {
```

Comment:
Just curiosity: why we use a classic function here and not an arrow one ?

Reply:
Just because we don't need to keep the `this`